src/index.ts

```diff
@@ -87,85 +96,100 @@ import {createLogger} from './logger';
  * @param {ConfigurationOptions} initConfiguration - The desired project/error
  *     reporting configuration.
  */
-function Errors(initConfiguration: ConfigurationOptions): void {
-  if (!(this instanceof Errors)) {
-    return new Errors(initConfiguration);
-  }
+export class Errors {
+  private _logger: Logger;
+  private _config: Configuration;
+  private _client: AuthClient;
+  report: (err: {}, request?: Request, additionalMessage?: string|{}, callback?: Callback|{}|string) => ErrorMessage;
```

Comment:
Is it possible to define `report` multiple times? ex. https://github.com/grpc/grpc-node/blob/9f0e496ad839eaeb5e09c349e742e9c844de3e25/packages/grpc-js-core/src/client.ts#L119

Reply:
Yes, since it is a change in the number of args it should work.

Reply:
To prevent merge conflicts for the PRs, I think it would be easier to update this in a follow-up PR.